### PR TITLE
feat(file): render file chips with proper link to download file

### PR DESCRIPTION
**Pull request by adrianschmidt**
fix: Lundalogik/crm-feature#2231

## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

File: src/components/chip-set/chip.types.ts [hunk 1]
```diff
@@ -49,4 +49,10 @@ export interface Chip<T = any> {
      * The value of the badge. Only valid for `filter`.
      */
     badge?: number;
+
+    /**
+     * If supplied, the chip will render an `<a>` element, using the supplied
```
Comment (jensgustafsson) [inline src/components/chip-set/chip.types.ts:54]: Isn't it always rendering an `<a>` element in this implementation?

Comment (adrianschmidt) [reply src/components/chip-set/chip.types.ts:54]: Yes, it is actually… I wrote this before I implemented it, and it turned out that rendering an `<a>` without a `href` was much easier that switching between rendering an `<a>` or a `<div>`, and it had the same result…

I'll update the description 👍

Comment (adrianschmidt) [reply src/components/chip-set/chip.types.ts]: ⚡

**Review by jensgustafsson — APPROVED** [review]
Nice!

I've tested it on iphone Safari and Chrome and on Mac OS Safari and Chrome

**Review by jensgustafsson — APPROVED** [review]

**Comment by adrianschmidt** [comment]
Note to self: Note sure we tested this with keyboard-navigation, so do that and fix if necessary…

**Pull request merged**